cli/command/image/build.go

```diff
@@ -114,6 +114,7 @@ func NewBuildCommand(dockerCli *command.DockerCli) *cobra.Command {
 	flags.BoolVar(&options.squash, "squash", false, "Squash newly built layers into a single new layer")
 	flags.SetAnnotation("squash", "experimental", nil)
 	flags.SetAnnotation("squash", "version", []string{"1.25"})
+	flags.SetAnnotation("squash", "os", []string{"windows"})
```

Comment:
`--squash` is not windows only, I think it was just being used as an example of an experimental flag.